src/tests/ftest/io/ior_hdf5_small.yaml

```diff
@@ -0,0 +1,51 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+    - server-C
+    - server-D
+  test_clients:
+    - client-E
+    - client-F
+    - client-G
+    - client-H
+timeout: 1800
+server_config:
+    name: daos_server
+    servers:
+        bdev_class: nvme
+        bdev_list: ["0000:81:00.0","0000:da:00.0"]
+        scm_class: dcpm
+        scm_list: ["/dev/pmem0"]
+pool:
+    mode: 146
+    name: daos_server
+    scm_size: 3000000000
+    nvme_size: 9000000000
+    svcn: 1
+    control_method: dmg
+container:
+    type: POSIX
+    properties: cksum:crc16,cksum_size:16384,srv_cksum:on
+    control_method: daos
+hdf5_vol:
+    plugin_path: /usr/lib64/mpich/lib
+ior:
+    client_processes:
+        np_16:
+            np: 16
+    test_file: daos:testFile
+    repetitions: 2
+    dfs_destroy: False
+    api: HDF5
```

Comment:
How does this differ to adding the HDF5 to the API list in ior_small?

Reply:
I created this separate test because ior_small as written fails on the first error and it may never test with hdf5 if there is an issue with dfs or mpiio or dfuse.    The separate test at least ensure that ior with hdf5 will get some test time.

Reply:
included ior -a HDF5 into ior small. Deleted the ior_hdf5_small test cases.